source/com.microsoft.tfs.client.clc/src/com/microsoft/tfs/client/clc/vc/Main.java

```diff
@@ -52,7 +55,29 @@ public void uncaughtException(final Thread t, final Throwable e) {
         int ret = ExitCode.FAILURE;
 
         try {
-            ret = main.run(args);
+            String[] commandArgs = args;
+
+            // Look for arg==@, this means that the user wants to type in the
+            // arguments one line at a time
+            // This was implemented to allow other programs to spin up the CLC
+            // and then pass arguments via stdin
+            if (args.length == 1 && args[0].equals("@")) { //$NON-NLS-1$
```

Comment:
Why not use the existing functionality, i.e. the Application#runCommandFile method?